services/jobrunner/justfile

```diff
@@ -0,0 +1,54 @@
+
+[private]
+default:
+  @just --list
+
+# show the status of the jobrunner service
+status:
+    sudo systemctl status -n0 jobrunner
+    python3 -m jobrunner.cli.flags get paused
+
+# start the jobrunner service
+start:
+    sudo systemctl start jobrunner
+
+# stop the jobrunner service
+stop:
+    sudo systemctl stop jobrunner
+
+# restart the jobrunner service
+restart:
+    sudo systemctl restart jobrunner
+
+# destroy containers & volumes for all running jobs
+prepare-for-reboot: stop
+    python3 -m jobrunner.cli.prepare_for_reboot
+
+# stop accepting new jobs
+pause:
+    python3 -m jobrunner.cli.flags set paused=true
+
+# start accepting new jobs
+unpause:
+    python3 -m jobrunner.cli.flags set paused=
+
+# show jobrunner logs (limited to 1000 lines) 
+logs lines="1000":
+    journalctl -xe -u jobrunner -n{{ lines }}
+
+# show jobrunner logs for a specific job_id
+logs-id job_id:
```

Comment:
don't think this is needed? we can just do `just logs | grep id`?

Reply:
it's not really needed, but I think it could be nice sugar in a workflow like

```bash
just jobrunner/logs-id 12345
# take a look
just jobrunner/job-retry 12345
```